### PR TITLE
fix: tag display correct when maxTagCount value is 0

### PR DESCRIPTION
**Pull request by grunmin**
allow maxTagCount to be 0, for some scenes like

### CODE REVIEW

**Comment by grunmin** [comment]
relate #258

**Comment by grunmin** [comment]
@yutingzhao1991 recreated and add test

File: src/Select.jsx [hunk 1]
```diff
@@ -1082,7 +1082,7 @@ export default class Select extends React.Component {
       let selectedValueNodes = [];
       let limitedCountValue = value;
       let maxTagPlaceholderEl;
-      if (maxTagCount && value.length > maxTagCount) {
+      if (Number.isInteger(maxTagCount) && value.length > maxTagCount) {
```
Comment (yesmeck) [inline src/Select.jsx]: IE does not support `Number.isInteger`,  I think check `maxTagCount  !== undefined` is enough.

Comment (grunmin) [reply src/Select.jsx]: well

Comment (yesmeck) [reply src/Select.jsx]: Don't need close, just change `Number.isInteger(maxTagCount)` to `maxTagCount !== undefined`

Comment (grunmin) [reply src/Select.jsx]: I thought I should create a new one cause I rewrite the last commit...

**Pull request closed without merging**